deployment_config/dev_vars.yml

```diff
@@ -1,12 +1,7 @@
-AUTH_BASE: https://uat.hsesinfo.org
-AUTH_CLIENT_ID: ((AUTH_CLIENT_ID))
-AUTH_CLIENT_SECRET: ((AUTH_CLIENT_SECRET))
 env: dev
 instances: 1
-NODE_ENV: production
```

Comment:
Again, because the value is the same in every env we can just define it in manifest.yml